processor/tasks/HelloWorld.py

```diff
@@ -0,0 +1,78 @@
+# # coding: utf-8
+
+# """
+# Simple law tasks that demonstrate how to build up a task tree with some outputs and dependencies.
+
+# The first task (FetchLoremIpsum) will download 1 of 6 different versions of a "lorem ipsum" text.
+# The next task (CountChars) determines and saves the frequency of every character in a json file.
+# After that, the count files are merged (MergeCounts). The last task (ShowFrequencies) illustrates
+# the "measured" frequencies and prints the result which is also sent as a message to the scheduler.
+# """
+
+
+import os
+import luigi
+import law
+from framework import Task, HTCondorWorkflow
+
+
+
+
+
+
+law.contrib.load("tasks")  # to have the RunOnceTask
+class LawBase(law.Task):
+    def local_path(self, *path):
+        # LAW_DATA_PATH is defined in setup.sh
+        parts = (os.getenv("ANALYSIS_DATA_PATH"),) + path
+        return os.path.join(*(str(p) for p in parts))
+
+    def local_target(self, *path, **kwargs):
+        return law.LocalFileTarget(self.local_path(*path), **kwargs)
+
+
+class HelloWorldSave(Task):
```

Comment:
do we need helloWorld code in this repo ?

Reply:
I use this series of tasks to have an example as a minimal workflow that includes remote execution and storage. It might be a good idea to rename it to something "example", but I would prefer to not remove it entirely.

Reply:
yes i like that, call it example and keep it !